web/public/locales/sv.json

```diff
@@ -249,11 +253,11 @@
   "aboutUs": {
     "groupName": "Om oss",
     "whoAreYou-question": "Vilka är ni?",
-    "whoAreYou-answer": "Electricity Map är utvecklad och förvaltad av <a href=\"https://electricitymap.org/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
+    "whoAreYou-answer": "Appen är utvecklad och förvaltad av <a href=\"https://electricitymap.org/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
```

Comment:
```suggestion
    "whoAreYou-answer": "Appen är utvecklad och förvaltad av <a href=\"https://electricitymaps.com/\" target=\"_blank\">Electricity Maps</a>, ett litet danskt nystartat företag. Vårt mål är att hjälpa mänskligheten att nå ett hållbart samhälle genom att samla in och tillgängliggöra klimatpåverkan från våra dagliga val.",
```

Reply:
It is multiple `electricitymap.org` in both `sv.json` and `en.json`, but no `electricitymaps.com`. 
This maybe should be changed in its own pull request if it is applied to all translations.

Reply:
sounds like a plan!